### DISCUSSION TITLE
Update PaperMC Java

### DISCUSSION THREAD

**Discussion by maxwhitepl** [GameServerManagers/LinuxGSM#3669] [Ideas]
With a standard Minecraft installation on V 1.17.1 with LinuxGSM
An error pops up due to the need for Java 16
> Minecraft 1.17 requires running the server with Java 16 or above. Download Java 16 (or above) from https://adoptopenjdk.net/

What can be bothersome for people if they want to quickly install the server and play quietly, they will be surprised how to fix it

![INDEX](https://i.imgur.com/dyFaGOd.jpg)

### TO DO
- Adding information that version higher than 1.17 Needs Java 16 which can be downloaded from [here](https://adoptopenjdk.net/)
- Adding Instruction for doc linuxgsm how download and fix this 

## Instruction to update for java:
- [LINK](https://paper.readthedocs.io/en/latest/java-update/index.html)

**Comment by johnoclockdk** [comment]
not a gud just update java to 16 or higher

**Reply by johnoclockdk** [reply]
+ it only for ubuntu 18.04,19.04,20.04

**Reply by maxwhitepl** [reply]
My point is that new installers from LinuxGSM server minecraft will not know what to do to run it

- I will create a guide for this in no time
